### DISCUSSION TITLE
Quick question!

### DISCUSSION THREAD

**Discussion by mikslikesplayingddlc** [Monika-After-Story/MonikaModDev#8601] [QA - tech support + general] [answered]
Heyhey, I have a question. How do I gift Monika?

**Comment by SpaceyFlakes** [comment, marked as answer]
I'm on windows so this may not work for you if you're on another device.
first off, ensure you can change the file type of things. You can do this by going to the computer control panel (change view by from category to large icons) and finding File Explorer Options, open it and go to the view tab under advanced settings scroll down until you find 'hide extensions for known file types' and make sure it's not checked.

If you can change the file type then go to the characters folder of your DDLC game dictionary, right click and choose New Text Document under 'new'. Rename the file and delete the entirety of the name, including .txt, and then rename it to the item you wish to gift Monika and .gift (eg, coffee.gift). If you've done it correctly it should ask if you wish to change the file's extension, press yes and either restart your game or wait until Monika notices.

Hope this helps!
